gcc/testsuite/rust.test/xfail_compile/xfail_compile.exp

```diff
@@ -14,19 +14,21 @@
 # along with GCC; see the file COPYING3.  If not see
 # <http://www.gnu.org/licenses/>.
 
-# This file was written by Rob Savoye. (rob@cygnus.com)
-# Modified and maintained by Jeffrey Wheat (cassidy@cygnus.com)
-
 #
-# Rust torture test suite.
+# Rust test suite.
+# These tests are expected to raise an error in the frontend
 #
 
 # Load support procs.
 load_lib rust-dg.exp
+load_lib torture-options.exp
 
 # Initialize `dg'.
 dg-init
 
+# Only use the default options
+set-torture-options [list {}]
+
```

Comment:
... remove that: this `set-torture-options` override will leak into other DejaGnu `*.exp` files executed after this one in the same `runtest` process...  (Yes, this is _ugly_, I don't like this behavior...)

Instead of `gcc-dg-runtest`, you can simply use `dg-runtest` below.

Reply:
Will do !